packages/core/email/admin/src/pages/Settings/index.js

```diff
@@ -109,130 +111,201 @@ const SettingsPage = () => {
+                    <Typography variant="delta" as="h2">
+                      {formatMessage({
+                        id: 'email.Settings.email.plugin.title.config',
+                        defaultMessage: 'Configuration',
+                      })}
+                    </Typography>
+                    <Typography>
+                      {formatMessage(
+                        {
+                          id: 'email.Settings.email.plugin.text.configuration',
+                          defaultMessage:
+                            'The plugin is configured through the {file} file, checkout this {link} for the documentation.',
+                        },
+                        {
+                          file: './config/plugins.js',
+                          link: (
+                            <DocumentationLink
+                              href="https://docs.strapi.io/developer-docs/latest/plugins/email.html"
+                              target="_blank"
+                              rel="noopener noreferrer"
+                            >
+                              {formatMessage({
+                                id: 'email.link',
+                                defaultMessage: 'Link',
+                              })}
+                            </DocumentationLink>
+                          ),
+                        }
+                      )}
+                    </Typography>
+                  </Flex>
+
+                  <Grid gap={5}>
+                    <GridItem col={6} s={12}>
+                      <TextInput
+                        name="shipper-email"
+                        label={formatMessage({
+                          id: 'email.Settings.email.plugin.label.defaultFrom',
+                          defaultMessage: 'Default sender email',
+                        })}
+                        placeholder={formatMessage({
+                          id: 'email.Settings.email.plugin.placeholder.defaultFrom',
+                          defaultMessage: "ex: Strapi No-Reply '<'no-reply@strapi.io'>'",
+                        })}
+                        disabled
+                        onChange={() => {}}
+                        value={data.settings.defaultFrom}
+                      />
+                    </GridItem>
+
+                    <GridItem col={6} s={12}>
+                      <TextInput
+                        name="response-email"
+                        label={formatMessage({
+                          id: 'email.Settings.email.plugin.label.defaultReplyTo',
+                          defaultMessage: 'Default response email',
+                        })}
+                        placeholder={formatMessage({
+                          id: 'email.Settings.email.plugin.placeholder.defaultReplyTo',
+                          defaultMessage: `ex: Strapi '<'example@strapi.io'>'`,
+                        })}
+                        disabled
+                        onChange={() => {}}
```

Comment:
Can we get rid of these empty onChange functions?

Reply:
With pleasure.